src/renderers/WebGLRenderer.js

```diff
@@ -1108,6 +1108,14 @@ function WebGLRenderer( parameters ) {
 		_currentMaterialId = - 1;
 		_currentCamera = null;
 
+		// onUpdate
+
+		scene.traverse( function( node ) {
+
+			node.onUpdate( _this, scene, camera );
```

Comment:
Could we have a way of registering objects for which `onUpdate()` needs to be called? That way, the renderer just needs to maintain a short array.